docs: add warning for Vite not watching files in WSL2 (#224)
motion-canvas/motion-canvas: packages/docs/docs/getting-started/quickstart.mdx

</CodeBlock>

Now save the file. Any changes you make are automatically picked up and
reflected in the preview. 

:::caution Vite on Windows Subsystem for Linux (WSL) 2

When running Vite on WSL2, file system watching does not work if a file is edited by Windows applications.

> For Vite to properly reload changed files, move the project folder into the WSL2 file system and use WSL2 applications to edit files. 

Reviewer: I think you could just say:
```suggestion
To fix this, move the project folder into the WSL2 file system and use WSL2 applications to edit files. 
```

> Accessing the Windows filesytem from WSL2 is slow, so this will improve performance.

Reviewer: ```suggestion
Accessing the Windows file system from WSL2 is slow, so this will improve performance.
```
